pkg/testutil/filters.go

```diff
@@ -6,10 +6,10 @@ import (
 	"berty.tech/weshnet/pkg/protocoltypes"
 )
 
-func TestFilterAppMetadata(t *testing.T, events <-chan *protocoltypes.GroupMetadataEvent) []*protocoltypes.AppMetadata {
+func TestFilterAppMetadata(t *testing.T, events <-chan *protocoltypes.GroupMetadataEvent) []*protocoltypes.GroupMetadataPayloadSent {
```

Comment:
TestFilterAppMetadata -> TestFilterGroupMetadataPayloadSent ?

Reply:
Nice catch. Thanks. I renamed it.